ThunderCloud/QuizBadgeCollectionCell.swift

```diff
@@ -36,9 +50,11 @@ open class QuizBadgeScrollerViewCell: CollectionCell {
     /// - Parameter atIndexPath: The IndexPath of the selected collection view cell
     open func handleSelectedQuiz(atIndexPath: IndexPath) {
         
-        guard let badges = badges else { return }
-        let badge = badges[atIndexPath.row]
-        guard let badgeId = badge.id else { return }
+        guard let badgeQuizzes = items as? [QuizBadge] else { return }
+        let badgeQuiz = badgeQuizzes[atIndexPath.row]
+        guard let badgeId = badgeQuiz.badge.id else { return }
+        let badge = badgeQuiz.badge
+        let quiz = badgeQuiz.quiz
```

Comment:
If quiz isn't used in that first if-statement (line 59-86), could the guard on 89 become `guard let quiz = badgeQuiz.quiz` instead?